hermes/templates/backup-deployment.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "hermes.name" . }}-store-backup
+  labels:
+    app: {{ template "hermes.name" . }}-store-backup
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "hermes.name" . }}-store-backup
+    spec:
+      affinity:
+{{ toYaml ( default .Values.global.affinity .Values.cronjob.affinity ) | indent 8 }}
+      containers:
+      - env:
+        - name: APP_ENVIRONMENT
+          valueFrom:
+            configMapKeyRef:
+              key: env
+              name: app
+        - name: REDIS_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              key: redis-password
+              name: hermes-store
+        - name: S3_URL
+          valueFrom:
+            secretKeyRef:
+              key: hermes-store-backup.s3-url
+              name: hermes-store-backup
+        - name: SLACK_CHANNEL
+          valueFrom:
+            secretKeyRef:
+              key: hermes-store-backup.slack-channel
+              name: hermes-store-backup
+        - name: SLACK_WEBHOOK_URL
+          valueFrom:
+            secretKeyRef:
+              key: hermes-store-backup.slack-url
+              name: hermes-store-backup
+        - name: AWS_ACCESS_KEY_ID
+          valueFrom:
+            secretKeyRef:
+              key: hermes-store-backup.aws-access-key-id
+              name: hermes-store-backup
+        - name: AWS_SECRET_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              key: hermes-store-backup.aws-secret-access-key
+              name: hermes-store-backup
```

Comment:
Use `{{ template "hermes.name" . }}-store-backup` exactly the same `name` as secret!